CODE_OF_CONDUCT.md

```diff
@@ -17,7 +17,7 @@ diverse, inclusive, and healthy community.
 Examples of behavior that contributes to a positive environment for our
 community include:
 
-- Demonstrating empathy and kindness toward other people
+- Demonstrating empathy and kindness towards other people
```

Comment:
I'm not sure about this one. I think this is a standardized text @eddiejaoude. Do we need to update this in every repo then?

Reply:
I have checked it on google the standardized text is **towards**. So I think it should be changed. Depends on maintainers what they think of it. That's just my point of view!